tests/test_shipments.py

```diff
@@ -479,8 +492,18 @@ def test_create_with_location(self):
         self.assertEqual(ship_to_location.name, parameters['_ship_to_location_name'])
         self.assertEqual(ship_to_location.geometry.coords, (12.0, 23.0))
 
+
+        test_settings.PROFILES_URL = 'DISABLED'
+        httpretty.register_uri(httpretty.GET,
+                               f"{test_settings.PROFILES_URL}/api/v1/wallet/{parameters['_shipper_wallet_id']}/",
+                               body=json.dumps({'good': 'good'}), status=status.HTTP_400_BAD_REQUEST)
+
+        response = self.client.post(url, one_location, content_type=content_type)
+        self.assertEqual(response.status_code, status.HTTP_202_ACCEPTED)
+
+        test_settings.PROFILES_URL = 'http://INTENTIONALLY_DISCONNECTED:9999'
```

Comment:
Instead of doing this, capture the original value of test_settings.PROFILES_URL in a temporary value, and use that to restore it. (that way if test_settings.PROFILES_URL is something other than http://INTENTIONALLY_DISCONNECTED:9999 in the future, this test won't break)